components/code/react-playground/react-playground.module.scss

```diff
@@ -1,118 +1,106 @@
+.column {
+  .codePane,
+  .previewPane {
+    width: 100%;
+    min-height: 10%;
+    max-height: 90%;
+  }
+  
+  .previewPane, .error {
+    border-bottom: none;
+    border-radius: 6px 6px 0 0;
+  }
+}
+
+.row {
+  .codePane,
+  .previewPane {
+    height: 100%;
+    min-width: 10%;
+    max-width: 90%;
+  }
+  
+  .previewPane, .error {
+    border-radius: 0 6px 6px 0;
+    border-left: none;
+  }
+}
+
```

Comment:
check this behavior with erez to make sure that its according to the product specs